Sources/FoundationEssentials/Decimal/Decimal+Conformances.swift

```diff
@@ -310,6 +307,56 @@ extension Decimal {
 
     @available(*, unavailable, message: "Decimal does not yet fully adopt FloatingPoint.")
     public mutating func formTruncatingRemainder(dividingBy other: Decimal) { fatalError("Decimal does not yet fully adopt FloatingPoint") }
+
+    public var nextUp: Decimal {
```

Comment:
Double checking: these new `public var` aren't new API, right? Did you move them from somewhere else?

Reply:
They are technically new API for SwiftFoundation but not for Foundation in general (it was only this way because initially we were trying to only bring the minimal of what we needed).